### DISCUSSION TITLE
Add new ddns provider CDMON.COM

### DISCUSSION THREAD

**Discussion by iperezme** [qdm12/ddns-updater#425] [General]
Hi!!
It would be possible to add the ddns update of the CDMON provider. On this website they explain how their DDNS works.

https://ticket.cdmon.com/es/support/solutions/articles/7000005922-api-de-actualizaci%c3%b3n-de-ip-del-dns-gratis-din%c3%a1mico/?set_locale=2&pk_vid=02ae8ef0c6125afd16752048900071b4

Thanks.
Israel Perez

**Comment by qdm12** [comment]
Created issue #746 to track this, closing this discussion.
